File: src/main/java/com/monkeyteam/monkeyutils/modules/Reload.java
package com.monkeyteam.monkeyutils.modules;

import com.monkeyteam.monkeyutils.MonkeyUtils;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;

public class Reload implements CommandExecutor {

    @Override
    public boolean onCommand(CommandSender commandSender, Command command, String s, String[] strings) {
        commandSender.sendMessage("Reloading config...");
        MonkeyUtils.getInstance().reloadConfig();
        commandSender.sendMessage("Config reloaded");
        return true;
    }

}

File: src/main/java/com/monkeyteam/monkeyutils/modules/Reload.kt
package com.monkeyteam.monkeyutils.modules

import com.monkeyteam.monkeyutils.MonkeyUtils.Companion.instance
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender

object Reload: CommandExecutor {
    override fun onCommand(sender: CommandSender, cmd: Command, label: String, args: Array<out String>): Boolean {
        sender.sendMessage("Reloading config...")
        instance.reloadConfig()
        sender.sendMessage("Config reloaded")
        return true
    }
}